cfme/infrastructure/virtual_machines.py

```diff
@@ -570,22 +587,39 @@ def _load(self):
         ems = appl_db['ext_management_systems']
         vms = appl_db['vms']
         hws = appl_db['hardwares']
+        guest_devices = appl_db["guest_devices"]
+
         hw_data = appl_db.session.query(ems, vms, hws).filter(
             ems.name == self.vm.provider.name).filter(
             vms.ems_id == ems.id).filter(
             vms.name == self.vm.name).filter(
             hws.vm_or_template_id == vms.id
         ).first().hardwares
+
         self.hw = VMHardware(
             hw_data.cpu_cores_per_socket, hw_data.cpu_sockets, hw_data.memory_mb, 'MB')
         hw_id = hw_data.id
 
+        # Network adapter
+        from cfme.infrastructure.provider.virtualcenter import VMwareProvider
```

Comment:
Please move this import to top

Reply:
Its inline import as not use in other part of Model page.